app/qml/InputStyle.qml

```diff
@@ -30,6 +30,8 @@ QtObject {
     property color labelColor: "#999999"
     property color secondaryFontColor: "#B3B3B3"
 
+    property color informationColor: "#86A3C3"
```

Comment:
what is used for? maybe in InputStyle it could be more based on usage (like (secondary)backgroundColor?)